packages/grafana-ui/src/components/Button/Button.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { Button } from './Button';
+
+describe('Button', () => {
+  it('should fire onClick when not disabled', async () => {
+    const onClick = jest.fn();
+    render(<Button onClick={onClick}>Click me</Button>);
+
+    const button = screen.getByRole('button');
+    await userEvent.click(button);
```

Comment:
We also [recommend](https://github.com/grafana/grafana/blob/3bb5a13275357df2233b183c64cb280a0bd1dcc0/contribute/style-guides/testing.md#testing-user-interactions) setting up user event instead of calling its methods directly.